hat/assets/js/cypress/integration/11 - submissions/list.spec.js

```diff
@@ -476,6 +476,56 @@ describe('Submissions', () => {
         });
     });
 
+    it.only('advanced settings should filter correctly', () => {
+        goToPage();
+        cy.get('#advanced-settings').click();
```

Comment:
This test generates an error
![Screenshot from 2023-07-15 19-22-56](https://github.com/BLSQ/iaso/assets/19631540/468e6cb2-87b5-4acc-950f-bd3f87fc03eb)

Reply:
also we should remove the `.only`

Reply:
@hakifran it works on my machine... But I'll change the `id` to use `data-test` instead.